develop/learning-paths/migration/articles/01-intro/01-from-a-monolith-to-a-modular-architecture.markdown

```diff
@@ -0,0 +1,124 @@
+
+What does all this mean? Has Liferay Portal simply been repackaged? No;
+it's been redeveloped into independent robust modules that work together to
+bring you a full-fledged portal. And on top of that, we've made the modules
+accessible in aggregations that are easier to release. We'll brief you on them
+now. 
+
+All of the suites run on Liferay Core. The core bootstraps the system, manages
+modules, and registers services. In addition to relying on the core, the suites
+rely on the Liferay Foundation suite for the building blocks familiar to Liferay
+users and developers, such as roles and permissions, users, groups, and user
+groups, sessions, login, and licensing, upgrades, clustering, and DAO, and
+front-end mainstays for themes, CSS, taglibs, and JavaScript. The Foundation
+suite's modules deploy on Liferay Core, as do all of the non-core modules.
+
+And then there are Liferay 7's other new suites that provide modules for the
+apps, frameworks, and APIs you've come to know and love as a Liferay platform
+developer.
+
+- **Liferay Web Experience**: Provides modules for apps such as Web Content
+Display, Asset Publisher, and Breadcrumbs, and provides powerful features and
+frameworks such as Application Display Templates, Tags, and Recycle Bin.
+Collectively they help you create, manage, and track content.
+
+- **Liferay Collaboration**: Users join together using the Message Boards, Wiki,
+and Blogs apps to share ideas and hash out details. It's Documents and Media
+Library continues to be your site's file store. And the suite's social apps and
+APIs enable you to track user activities and attribute social equity.
+
+- **Liferay Forms and Workflow**: Boosts productivity with its Calendar, Forms,
+and Workflow app modules. Its Dynamic Data Mapping framework helps you bring new
+meaning to your structured content.
+
+Now, you could take the core and its suites to be a simple shuffling around of
+things. But it's much more than that. For one, the core and each of the suites
+have their own release cycles. That means, you can grab the latest Liferay Core
+release and not burden yourself with updating any of the other suites. If a new
+version of new Liferay Collaboration has a hot new feature or API improvement
+you want, you can focus on upgrading to that new version of Liferay
+Collaboration. Even at the module aggregation level, you start to see the
+freedom.
+
+But wait a minute! How does an app in one suite use an app in another? For
+example, the Web Content Display in Web Experience uses the Comments framework
+that's in Collaboration. How does that work? Is there tight coupling between
+them? To answer these questions, we need to get a deeper understanding of how
+Liferay 7 embraces modularity. We'll explain how we've implemented modularity
+and compare Liferay 7 modules with Liferay 6 plugins. 
```

Comment:
Haven't reviewed anything below this point, per your statement that you were rewriting.